packages/jest-haste-map/src/index.js

```diff
@@ -49,6 +49,7 @@ type Options = {
   retainAllFiles: boolean,
   roots: Array<string>,
   useWatchman?: boolean,
+  throwOnModuleCollision?: boolean,
```

Comment:
please alphabetize the items in this list, we try to be consistent wherever we can :)